src/multiplay.cpp

```diff
@@ -193,22 +193,27 @@ bool multiplayerWinSequence(bool firstCall)
 	return true;
 }
 
-// ////////////////////////////////////////////////////////////////////////////
-// ////////////////////////////////////////////////////////////////////////////
-// MultiPlayer main game loop code.
-bool multiPlayerLoop()
+int countJoiningHumans()
 {
-	UDWORD		i;
-	UBYTE		joinCount;
-
-	joinCount = 0;
-	for (i = 0; i < MAX_PLAYERS; i++)
+	int joinCount = 0;
+	for (int i = 0; i < MAX_PLAYERS; i++)
 	{
 		if (isHumanPlayer(i) && ingame.JoiningInProgress[i])
 		{
```

Comment:
@KJeff01 I used this snippet, whereas #526 uses `NetPlay.players[Inc].allocated`. Is that better? I could steal that piece from #526 (probably resulting in automatically resolved merge conflict).

Reply:
isHumanPlayer() ultimately checks .allocated which is true for humans and false for AI (and does some player index checking as well).

Reply:
OK, I stole `NET_numHumanPlayers()` from #526.